src/base_directories.rs

```diff
@@ -916,15 +916,16 @@ mod test {
             .to_owned()
     }
 
+    #[cfg(test)]
```

Comment:
The entire module is wrapped in `#[cfg(test)]`. What problem does this solve?

Reply:
sorry about that, I didn't review the code carefully